python/paddle/fluid/dygraph/dygraph_to_static/program_translator.py

```diff
@@ -261,19 +263,20 @@ def __init__(self, exe=None, place=None):
         else:
             self._exe = exe
         self._program_cache = ProgramCache()
+        self._optimizer_info = None
         self._optimizer = None
-        self._already_minimized = False
+        self._loss_name = None
         # Once main_program is changed, should run startup_program.
         self._need_startup = True
 
     def get_output(self, dygraph_func, *args, **kwargs):
         """
-        Returns the output tensors for dygraph function and its arguments
+        Return the output tensors for dygraph function and its arguments
```

Comment:
Returns, supports is more common in SWE, but you don't have to change in this PR

Reply:
I will change it back in https://github.com/PaddlePaddle/Paddle/pull/23652